deps/rabbitmq_mqtt/src/rabbit_mqtt_processor.erl

```diff
@@ -509,6 +511,17 @@ process_request(?SUBSCRIBE,
                                        reason_codes = lists:reverse(ReasonCodesRev)}},
     _ = send(Reply, State1),
     case hd(ResultRev) of
+        {error, access_refused} ->
+            %% If disconnect_on_unauthorized is false, do not disconnect the client,
+            %% send retained messages for the topics to which the client could successfully subscribe.
+            %% Otherwise, disconnect the client, treat the subscription failure.
+            case application:get_env(rabbitmq_mqtt, disconnect_on_unauthorized, true) of
```

Comment:
You already configured the default of `true` in the `Makefile`. So, you can omit the default `true` in this line. It's better to configure the defaults for configuration in just one place instead at multiple places throughout the code base.

Reply:
You can use here the `?APP_NAME` macro.